pkg/server/server.go

```diff
@@ -53,6 +52,22 @@ const (
 	grpcDialTimeout = 5 * time.Second
 )
 
+func ServeUI(ctx context.Context, logger log.Logger, cfg UIConfig) {
+	isUIPortNotExits := cfg.Port == 0
+	if isUIPortNotExits {
+		logger.Error("ui server failed: no port specified")
+		return
+	}
+
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		http.ServeFile(w, r, "ui/dist/ui/index.html")
```

Comment:
I ran the `make build` command but `ui/dist` folder is empty. getting 404 error in browser.
`make ui` also gave error as packages are not installed.
[
<img width="416" alt="Screenshot 2024-01-17 at 9 19 06 PM" src="https://github.com/raystack/frontier/assets/9323589/8e6577de-f484-417d-9b4b-78b75a524c69">
](url)